template.json

```diff
@@ -0,0 +1,53 @@
+{
+  "variables": {
+    "token":"",
+    "image_name": "hasura-marketplace-snapshot-{{timestamp}}"
+  },
+  "builders": [
+    {
+      "type": "digitalocean",
+      "api_token": "{{user `token`}}",
+      "image": "docker-18-04",
+      "region": "blr1",
+      "size": "s-1vcpu-1gb",
+      "ssh_username": "root",
+      "snapshot_name": "{{user `image_name`}}"
+    }
+  ],
+  "provisioners": [
+    {
+      "type": "file",
+      "source": "files/etc/",
+      "destination": "/etc/"
+    },
+    {
+      "type": "file",
+      "source": "files/var/",
+      "destination": "/var/"
+    },
+    {
+      "type": "shell",
+      "inline": [
+        "cloud-init status --wait"
+      ]
+    },
+    {
+      "type": "shell",
+      "inline": [
+        "rm -rvf /var/lib/apt/lists/*",
```

Comment:
There is an apt-get upgrade happening in the cleanup script. Let us remove that?